### PR TITLE
fix: make it more difficult for users to override test UI CSS

### LINKED ISSUE #269: Buttons move with the button css selector
#### Issue Description

In codepen if you do something like:

button {
margin-left: 100px;
}

it will move the buttons over 100px in the test suite buttons.

#### Browser Information

* Browser Name, Version: Chromium 63
* Operating System: Linux Mint 18.3
* Mobile, Desktop, or Tablet: Desktop

#### Your Code / Link to Your Pen

```
I adjusted for this by just using an id selector but figured it was worth mentioning. 
 
 
```

#### Screenshot

test1

### PR DESCRIPTION
**Pull request by tchaffee**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [x] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

- [x] Closes currently open issue: Closes #269 

#### Description
Made sure the most common properties are specified on the parent div for the test UI and then add a CSS rule so that all those properties are inherited by all children elements. This is a little heavy handed and required that we unset or specify those properties in some children elements, but it should go a long way towards making our CSS more specific than an element override in the user CSS.

### CODE REVIEW

**Comment by tchaffee** [comment]
@tbushman If you want to test this, I created a CodePen that tries to override element properties and you can see that the test UI is immune. 

https://codepen.io/tchaffee/pen/aYyPRE

Note the following CSS in the CodePen that tries to set properties:

```
div {
  margin-left: 300px;
  background-color: black;
}

span {
  margin-left: 300px;
  background-color: black;
}

button {
  margin-left: 100px;
  background-color: black;
}

label {
  margin-left: 100px;
  font-family: cursive;
  width: 500px;
  background-color: black;
}
```

File: src/stylesheets/fcc-test-ui.css [hunk 1]
```diff
@@ -1,71 +1,93 @@
-
 #fcc_foldout_menu_inner {
```
Comment (tbushman) [inline src/stylesheets/fcc-test-ui.css]: This needs a transparent background color to cancel out the yellow.

**Comment by tchaffee** [comment]
@tbushman Thanks for the review. I fixed the problem with the h1 element and the made the other change you suggested. I think this is ready for merge now unless you have other suggestions.

**Comment by tbushman** [comment]
jQuery upgrade included. Tested. This closes also https://github.com/freeCodeCamp/freeCodeCamp/issues/16808#issuecomment-370609617 LGTM!

**Pull request merged**